docs/contributing.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Contribution section belongs to Github README.

Reply:
OK.